src/components/dashboard/dashboardItem/DataConsumption.js

```diff
@@ -44,6 +44,22 @@ ChartJS.register(
 );
 
 const options = (usage, quota, date, distance, title, theme, t) => {
+    let quotaGiB = quota / constants.ONE_GiB;
+    let divisor =
+        quotaGiB % 4 === 0
+            ? 4
+            : quotaGiB % 3 === 0
+            ? 3
+            : quotaGiB % 5 === 0
+            ? 5
+            : quotaGiB % 2 === 0
+            ? 2
+            : 4;
+    let maxTicks = 6;
+    let stepSize =
+        usage <= (quota / divisor) * maxTicks
+            ? Math.ceil(quota / divisor)
```

Comment:
I'm not 100% sure this `Math.ceil` is the right thing to do. But, it'll only come up if the `divisor` calculation falls through to the default value of 4, which only happens if the quota's prime factorization doesn't include anything below 7, and it seems unlikely that'd happen (in fact, I'd say it's pretty unlikely we'd do any that aren't multiples of 100, which is already a multiple of 2,4, and 5).

Reply:
well, or if the quota's not a multiple of a GiB, but that seems even less likely